tf_files/aws/sftp/cloud.tf

```diff
@@ -0,0 +1,84 @@
+terraform {
+  backend "s3" {
+    encrypt = "true"
+  }
+}
+
+provider "aws" {}
```

Comment:
awesome - 
* please also add a `manifest.json` to bump up to terraform12 - see `aws/data_bucket/`
* add a test case to `terraformTest.sh` to take the script through `gen3 tfplan`: 
    - https://github.com/uc-cdis/cloud-automation/blob/master/gen3/test/terraformTest.sh#L82
    - https://github.com/uc-cdis/cloud-automation/blob/master/gen3/test/terraformTest.sh#L327
    - `gen3 testsuite --filter test_databucket_workspace`